docs/mkdocs.yml

```diff
@@ -153,6 +153,7 @@ nav:
         - 'IPv6': config/advanced/ipv6.md
         - 'Podman': config/advanced/podman.md
         - 'Dovecot Master Accounts': config/advanced/dovecot-master-accounts.md
+        - 'iOS Mail Push Support': config/advanced/ios-mail-push-support.md
```

Comment:
Since this isn't about official DMS config, it might be better placed in the similar Examples section of our docs (_just below this in the config_)?

If there is an area in our other docs that would benefit raising user awareness to this new content, you could direct them to that examples page too.

Reply:
I can also move this to the examples section. Below tutorials? Or do you consider this a usecase?

Reply:
Either is fine :)

Perhaps is a use-case since its more of a niche feature to configure.

Reply:
Moved to use-cases.